boxoffice/views/admin_item.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+
+from .. import app, lastuser
+from coaster.views import load_models, requestargs
+from baseframe import _
+from boxoffice.models import db, Organization, ItemCollection, Item
+from boxoffice.views.utils import api_error, api_success
+from utils import xhr_only
+
+
+@app.route('/admin/o/<org>/items')
+@lastuser.requires_login
+@xhr_only
+@load_models(
+    (Organization, {'name': 'org'}, 'organization'),
+    permission='org_admin'
+    )
+@requestargs('search')
+def items(organization, search=None):
+    if search:
+        filtered_items = db.session.query(Item, ItemCollection).filter(
+            ItemCollection.organization == organization).filter(
+            Item.title.ilike('%{query}%'.format(query=search))).join(Item.item_collection).options(
```

Comment:
To enable full text indexing, you'll need a migration, plus a custom `before_create` event listener for SQLAlchemy. Look at Hasjob for an example of how to do it.